cmd/authenticator/main.go

```diff
@@ -18,20 +18,20 @@ var infoLogger = authenticator.InfoLogger
 func main() {
 	var err error
 
-	// Parse any flags for client cert / token paths, and set default values if not passed
-	clientCertPath := flag.String("c", "/etc/conjur/ssl/client.pem",
```

Comment:
yes, ultimately. not only these vars but all of them. i'd rather do that properly with a thourough design because it raises some question (i.e which input is higher in priority if both inputs are given).

i think it's better to have all the inputs as env vars and then re-think the whole thing. here is the ticket: https://github.com/cyberark/conjur/issues/1142

Reply:
I'd rather not have half-baked features in master (adds complexity and requires more tests). if these parameters are not used as cmd inputs at the moment i'd rather fully design this and re-add them later with all the rest.
@izgeri @sgnn7 @jonahx @doodlesbykumbi do you know of any usage of `clientCertPath` & `tokenFilePath` via args or do we always use the default value?

Reply:
@orenbm Given that the image for this client is out in the world at large, perhaps there are users leveraging the flags already. I'm not aware of any usage of the flags on out side.

Reply:
ok i re-added this ability. thanks!